docs/input/vue.md

```diff
@@ -0,0 +1,81 @@
+## Import
+
+#### Use in entire app
+
+```js
+import { Forms } from '@warp-ds/vue'
+app.use(Forms)
+```
+
+#### Use in one component and special imports
+
+```js
+import { wInput } from '@warp-ds/vue'
+```
+
+## Usage
+
+```js
+<w-input label="A label" hint="A hint" v-model="model" />
+```
+
+## Props
+
+The props documented below have defaults set or are unique to this component, all typical HTML5 attributes are valid props. See Field for additional props.
```

Comment:
> See Field for additional props

Unless it's obvious where to look and I just can't see it, could we try and have a link that points to the right place in the docs?

Reply:
This doesn't exist yet. As in Vue the docs are structured in another way. So I would add a link but I don't have any content for Field yet. We can maybe have a separate talk about how to structure docs for these field parts

Reply:
Added props here https://github.com/warp-ds/tech-docs/pull/10/commits/2cb3dcef8bafaa05fea16efdb114db81ad5bc38c but we will handle Field separately. In a separate task

Reply:
https://trello.com/c/1B5lBw01/185-what-about-field-component-it-is-mentioned-in-input-textarea-and-only-exists-in-vue-how-should-we-document-it-or-not-document-it

Reply:
Amazing!